processor/spanmetricsprocessor/processor.go

```diff
@@ -410,11 +445,13 @@ func (p *processorImp) aggregateMetricsForSpan(serviceName string, span pdata.Sp
 	p.updateLatencyMetrics(key, latencyInMilliseconds, index)
 	p.updateLatencyExemplars(key, latencyInMilliseconds, span.TraceID())
 	p.lock.Unlock()
+	stats.Record(context.TODO(), spanProcessedCount.M(int64(1))) // count 1 here for a processed span.
 }
 
 // updateCallMetrics increments the call count for the given metric key.
 func (p *processorImp) updateCallMetrics(key metricKey) {
 	p.callSum[key]++
+	stats.Record(context.TODO(), uniqueTimeSeriesCount.M(int64(len(p.callSum))))
```

Comment:
Can this be done somewhere else that isn't inside of a loop? Tracking the metric here will add a performance hit